sdk/src/types/block/payload/signed_transaction/transaction_id.rs

```diff
@@ -18,3 +18,11 @@ impl TransactionId {
         OutputId::new(self, index)
     }
 }
+
+crate::impl_id!(
+    /// The hash of a transaction commitment and output commitment which is used to create
+    /// [`SignedTransactionPayload`](crate::types::block::payload::SignedTransactionPayload).
```

Comment:
Wrong comment

Reply:
Signing hash is just the hash of the serialised bytes

Reply:
oh I got it backwards oops